projboard/tests.py

```diff
@@ -0,0 +1,3 @@
+from django.test import TestCase
```

Comment:
Please comment or remove this line,
We are not using TestCases for testing.

Reply:
I already comment this line in the second commit (Fix ci errors)